impl/client/src/main/javascript/web/pentaho/type/_type.js

```diff
@@ -901,9 +887,9 @@ define([
       /**
        * Gets the style classes of this and any base types.
        *
-       * Do **NOT** modify the returned array.
+       * Do **not** modify the returned array.
        *
-       * @type string[]
+       * @type {string[]}
```

Comment:
Whatever we write, Js Doc outputs using the generic syntax. I think that the non-generic syntax cannot represent nested types, like this one: `Map.<string, Object>[]`, which can only be written as `Array.<Map.<string, Object>>`. Even so, I'd prefer not being strict with this, and allow both, as the non-generic syntax is easier, shorter and more rooted.